examples/classes/b.example

```diff
@@ -5,11 +5,11 @@ class Rectangle {
   }
 
   get color() {
-    return this.color;
+    return this._color;
   }
 
   set color(c) {
-    this.color = c;
+    this._color = c;
```

Comment:
Evaluating `this.color` inside the `color` getter method will invoke `color` getter method.  Infinite loop.